### PR TITLE
Implement class and alt attributes for ImageShortCode

### LINKED ISSUE #6819: Shortcodes with parameters
I need to add a custom class to the image rendered by the ImageShortCode.  I was hoping it could be achieved by this
[image class="image-fluid"] path of image [/image].

But it doesn't work. Can anyone suggest any other approaches ?

### PR DESCRIPTION
**Pull request by KshitizGIT**
Added support for attributes 'class' and 'alt' in ImageShortcodeProvider. Also, added supporting test cases.
This fixes #6819 and #6745 .

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/OrchardCMS/OrchardCore?pullRequest=6838) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/OrchardCMS/OrchardCore?pullRequest=6838) before we can accept your contribution.<br/><br/>:x: KshitizGIT [sign now](https://cla.dotnetfoundation.org/OrchardCMS/OrchardCore?pullRequest=6838)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/OrchardCMS/OrchardCore?pullRequest=6838) it.</sub>

File: src/OrchardCore.Modules/OrchardCore.Media/Shortcodes/ImageShortcodeProvider.cs [hunk 1]
```diff
@@ -95,10 +98,20 @@ public ValueTask<string> EvaluateAsync(string identifier, Arguments arguments, s
...
+                if (altText != null)
+                {
+                    altText = "alt=\"" + altText + "\" ";
+                }
+
                 content = QueryHelpers.AddQueryString(content, queryStringParams);
             }
 
-            content = "<img src=\"" + content + "\">";
+            content = "<img " + altText + className + "src=\"" + content + "\">";
```
Comment (hishamco) [inline src/OrchardCore.Modules/OrchardCore.Media/Shortcodes/ImageShortcodeProvider.cs:114]: Can we use string interpolation here?

Comment (deanmarcussen) [reply src/OrchardCore.Modules/OrchardCore.Media/Shortcodes/ImageShortcodeProvider.cs:114]: No, this is fine, and better for performance

**Comment by KshitizGIT** [comment]
Thank you @deanmarcussen .  😀

File: src/OrchardCore.Modules/OrchardCore.Media/Shortcodes/ImageShortcodeProvider.cs [hunk 2]
```diff
@@ -95,10 +98,20 @@ public ValueTask<string> EvaluateAsync(string identifier, Arguments arguments, s
                     queryStringParams.Add("rmode", mode);
                 }
 
+                if (className != null)
+                {
+                    className = "class=\"" + className + "\" ";
```
Comment (sebastienros) [inline src/OrchardCore.Modules/OrchardCore.Media/Shortcodes/ImageShortcodeProvider.cs:103]: Shouldn't it be HTML encoded?

Comment (sebastienros) [reply src/OrchardCore.Modules/OrchardCore.Media/Shortcodes/ImageShortcodeProvider.cs:103]: But then would need to check "-" is not encoded by the default encoder.

Comment (KshitizGIT) [reply src/OrchardCore.Modules/OrchardCore.Media/Shortcodes/ImageShortcodeProvider.cs:103]: On Line 115, content =  _htmlSanitizerService.Sanitize(content)  will encode the className

**Comment by hishamco** [comment]
@agriffard could you please add @KshitizGIT as contributor if it's not listed yet

Thanks

File: src/OrchardCore.Modules/OrchardCore.Media/Shortcodes/ImageShortcodeProvider.cs (hunk 2 above)
Comment (sebastienros) [reply src/OrchardCore.Modules/OrchardCore.Media/Shortcodes/ImageShortcodeProvider.cs:103]: Awesome, missed it, thanks.

**Pull request merged**